server.js

```diff
@@ -4877,6 +4877,61 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Jenkins coverage integration (jacoco)
+camp.route(/^\/jenkins(?:-ci)?\/j\/(http(?:s)?)\/([^/]+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var scheme = match[1];  // http(s)
```

Comment:
I suggest using the ES6 `let` and `const` keywords to declare variables. We still have a lot of code exclusively making use of `var`, but we try to update things as we go along. 😉